nmtwizard/preprocess/tu.py

```diff
@@ -120,8 +122,21 @@ def adjust_alignment(self, side_idx, start_idx, tok_num, new_tokens=None, part =
                     new_alignment.add((opp_side_tok_idx, side_tok_idx))
                 else:
                     new_alignment.add((side_tok_idx, opp_side_tok_idx))
+
             self.__alignments[part] = new_alignment
 
+    def insert_aligned_tokens(self, src_start_idx, tgt_start_idx, src_nb_inserted_tokens, tgt_nb_inserted_tokens, part = 0):
```

Comment:
Maybe just rename it to 'insert_alignments' or something like that, to avoid confusion